veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,185 @@
+# VEP #20: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+This proposal outlines the enhancements implemented in passt and how KubeVirt will integrate with passt to benefit from them:
+- vhost-user protocol
+- Seamless live migration of TCP sessions
+
+## Motivation
+
+Currently, KubeVirt does not support all networking requirements expected by users:
+- Seamless network connectivity on migration
+- Observability/Service Mesh support
```

Comment:
Could you please provide an example of an observability system we currently cannot use?

Reply:
I know that there are folks who would like to use https://github.com/stackrox/collector together with KubeVirt, and they need to use passt for that, because that project relies on observation of system calls issues by applications. I can ask around if you need more specific details, let me know.